astropy/coordinates/tests/test_geodetic_representations.py

```diff
@@ -2,16 +2,65 @@
 
 """Test geodetic representations"""
 import pytest
-from numpy.testing import assert_array_equal
 
+from numpy.testing import assert_array_equal
 from astropy import units as u
+from astropy.units import allclose as quantity_allclose
+from astropy.coordinates.geodetic import BaseGeodeticRepresentation
+
 from astropy.coordinates.earth import (
     GRS80GeodeticRepresentation,
     WGS72GeodeticRepresentation,
     WGS84GeodeticRepresentation,
 )
 from astropy.coordinates.representation import CartesianRepresentation
-from astropy.units import allclose as quantity_allclose
+from astropy.units.tests.test_quantity_erfa_ufuncs import vvd
+
+
+def test_custombasegeodetic_error():
+    # Test incomplete initialization.
+    msg = (
+        "type object 'BaseGeodeticRepresentation' has no attribute '_equatorial_radius'"
+    )
+    s1 = CartesianRepresentation(
+        x=[1, 3000.0] * u.km, y=[7000.0, 4.0] * u.km, z=[5.0, 6000.0] * u.km
+    )
+
+    custom_geodetic = BaseGeodeticRepresentation
```

Comment:
You should actually define a subclass, otherwise you are setting attributes on the real class which can come back to bite you. I.e.,
```
class CustomGeodetic(BaseGeodeticRepresentation):
    _flattening = 0.075 * u.dimensionless_unscaled
````